src/Campaigns/resources/admin/components/CampaignDetailsPage/Components/ColorControl/index.tsx

```diff
@@ -0,0 +1,100 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import {useCallback, useState} from 'react';
```

Comment:
Import these from `@wordpress/element`

Reply:
https://lw.slack.com/archives/C04SLRDD9CK/p1696539982208569

Reply:
Hmm interesting. I think more than half of our code is using `@wordpress/element`. Anyway, proceed...?

Reply:
@pauloiankoski let's move this to QA